### PR TITLE
Config options for Birmingham Baskerville slurm

### PR DESCRIPTION
**Pull request by jnywong**
Hi, 

Adding the Baskerville cluster (Univeristy of Birmingham, UK), to the snippets library, as per instructions in the README.

Jenny

### CODE REVIEW

File: _includes/snippets_library/Birmingham_Baskerville_slurm/_config_options.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  prompt: "[user@laptop ~]$"
+  bash_shebang: "#!/usr/bin/env bash"
+
+remote:
+  name: "Baskerville"
+  login: "login.baskerville.ac.uk"
+  host: "bask-pg0310u18a.cluster.baskerville.ac.uk"
+  node: "bask-pg"
+  location: "University of Birmingham, UK"
+  homedir: "/bask/homes/y/yourUsername"
```
Comment (jnywong) [inline _includes/snippets_library/Birmingham_Baskerville_slurm/_config_options.yml:32]:
```suggestion
  homedir: "/bask/homes/y/"
```

Get rid of extra username subdir in path

File: _includes/snippets_library/Birmingham_Baskerville_slurm/_config_options.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  prompt: "[user@laptop ~]$"
+  bash_shebang: "#!/usr/bin/env bash"
+
+remote:
+  name: "Baskerville"
+  login: "login.baskerville.ac.uk"
+  host: "bask-pg0310u18a.cluster.baskerville.ac.uk"
+  node: "bask-pg"
+  location: "University of Birmingham, UK"
+  homedir: "/bask/homes/y/"
```
Comment (jnywong) [inline _includes/snippets_library/Birmingham_Baskerville_slurm/_config_options.yml:32]:
```suggestion
  homedir: "/bask/homes/y"
```

Remove trailing slash

**Comment by tkphd** [comment]
Hi @jnywong,

Thank your for contributing your snippet library! This is great!

It looks like this PR was made from the default branch (`gh-pages`) on the @baskerville-hpc fork of this repository. Unfortunately, this means that the fork has diverged from this repository, and makes for some maintenance headaches -- for example, resolving the _This branch is out-of-date with the base branch_ error. We strongly recommend a [branching workflow for PRs](https://github.com/carpentries-incubator/hpc-intro/blob/gh-pages/CONTRIBUTING.md#using-github), which will compartmentalize changes that have not yet been merged and allow for easier synchronization between forks.

This situation is recoverable, but involves renaming and deleting branches, which carries a high risk of losing your changes. Please let me know (here or on [#hpc-carpentry](https://github.com/carpentries-incubator/hpc-intro/blob/gh-pages/MAINTENANCE.md#slack)) if you'd like a second set of eyes on your terminal as you work through it.

Thank you,
Trevor

**Comment by jnywong** [comment]
Hi @tkphd,

Apologies for my mistake, I'll take a look again and let you know if I need any help. Thanks for your very detailed advice.

Best wishes,
Jenny

**Comment by jnywong** [comment]
Closed and replaced by https://github.com/carpentries-incubator/hpc-intro/pull/421

**Pull request closed without merging**